test/data/addReactions_err_test.yaml

```diff
@@ -0,0 +1,169 @@
+description: |-
+  This is the file for testing Plog errors
+
+generator: ck2yaml
+input-files: [mech.inp, therm.DAT, trans.dat]
+cantera-version: 2.5.0a2
+date: Mon, 29 Jun 2020 14:13:58 +0000
+
+units: {length: cm, time: s, quantity: mol, activation-energy: cal/mol}
+
+phases:
+- name: gas
+  thermo: ideal-gas
+  elements: [H, O, C, N, Ar, He]
+  species: [CH3, CO, CO2, CH3O, CH2CHOO, H2O2, OH, H2O, HO2]
+  kinetics: gas
+  transport: mixture-averaged
+  state: {T: 300.0, P: 1 atm}
+
+species:
+
+- name: CO2
+  composition: {C: 1, O: 2}
+  thermo:
+    model: NASA7
+    temperature-ranges: [200.0, 1000.0, 6000.0]
+    data:
+    - [2.356813, 8.9841299e-03, -7.1220632e-06, 2.4573008e-09, -1.4288548e-13,
+      -4.8371971e+04, 9.9009035]
+    - [4.6365111, 2.7414569e-03, -9.9589759e-07, 1.6038666e-10, -9.1619857e-15,
+      -4.9024904e+04, -1.9348955]
+  transport:
+    model: gas
+    geometry: linear
+    well-depth: 244.0
+    diameter: 3.763
+    polarizability: 2.65
+    rotational-relaxation: 2.1
```

Comment:
Also this, and the other transport properties as well. I'm actually surprised you can specify a transport model in the phase, since a few species don't have transport parameters.

Reply:
The reason this doesn't cause a problem is because loading the file results in an error in creating the `Kinetics` object before it gets to trying to create a `Transport` object.